### PR TITLE
fix assets for launch

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: locales/en-US/experiments.ftl [hunk 1]
```diff
@@ -24,7 +24,6 @@ activitystreamContributors11Title = Software Engineer
 activitystreamContributors12Title = Senior UX Designer
 cliqzDescription = Get search results faster. Cliqz shows suggestions right below the URL bar as you type. All while maintaining your privacy.
 cliqzIntroduction = Get search results faster, directly in the browser. Start typing and see suggestions from around the Web in real time, right below the URL bar. And because Cliqz avoids collecting personally identifiable data, or building user profiles, your results are more private.
-cliqzPrivacypreamble = The Cliqz Test Pilot experiment is brought to you through collaboration between Cliqz GmbH, and Mozilla. Cliqz is designed to protect your privacy and avoid collecting sensitive information that can be used to create user profiles. However, anonymizing data like this is difficult, and it may still be possible to identify specific individuals.
```
Comment (johngruen) [inline locales/en-US/experiments.ftl]: @clouserw this wasn't in the build dir...not sure if it needs to be, but i copy-pastaed the build en-us experiments.ftl to pick up the new changes and this was missing

File: locales/en-US/experiments.ftl [hunk 2]
```diff
@@ -54,7 +76,7 @@ nomore404sSubtitle = Powered by the Wayback Machine
 nomore404sDescription = Tired of dead ends on the web? We'll let you know when there's a saved version of what you're looking for on the Internet Archive's Wayback Machine.
 nomore404sIntroduction = <p>This one is simple. If you hit a 404 page while browsing the web, we will let you know if the content you're looking for is archived on the Internet Archive's Wayback Machine.</p>
 nomore404sMeasurements0 = We collect basic usage on how many times you encounter a Page Not Found error (code 404), how many times a cached version of that page exists from Archive.org, and how many times you choose to view the cached version.
-nomore404sMeasurements1 = To provide cached versions of pages, we send 404 error page URLs to Archive.org. <a href="https://archive.org/" target="_blank">Archive.org</a> discloses its privacy policy <a href="https://archive.org/about/terms.php" target="_blank">here</a>.
+nomore404sMeasurements1 = To provide cached versions of pages, we send 404 error page URLs to Archive.org. <a href="https://archive.org/" target="_blank">Archive.org</a> discloses its privacy policy <a href="https://archive.org/about/terms.php" target="_blank">here</a>.</li>
```
Comment (johngruen) [inline locales/en-US/experiments.ftl]: @clouserw i don't know why there's an LI here, again, this is pure copypasta. maybe a formatting error in the yaml

File: locales/en-US/experiments.ftl [hunk 3]
```diff
@@ -119,7 +177,7 @@ trackingprotectionEolwarning = We will automatically disable the Tracking Protec
 universalsearchDescription = Get recommendations for the best sites on the web as you type in the Awesome Bar.
 universalsearchIntroduction = <p>Get to the best of the Web faster with Universal Search. Just type a few characters into your Awesome Bar, and the most popular sites, people and Wikipedia articles will appear for your browsing pleasure.</p>
 universalsearchMeasurements0 = We collect basic usage data regarding how you interact with the Awesome Bar (address bar), such as the number of characters typed, the types of results that you click, and their position in the results list.
-universalsearchMeasurements1 = If you previously configured Firefox to display search suggestions in the Search Bar, installing Universal Search will enable <a href="https://www.mozilla.org/privacy/firefox/#searchsuggestions" target="_blank">search suggestions</a> in the Awesome Bar.
+universalsearchMeasurements1 = If you previously configured Firefox to display search suggestions in the Search Bar, installing Universal Search will enable <a href="https://www.mozilla.org/privacy/firefox/#searchsuggestions" target="_blank">search suggestions</a> in the Awesome Bar.</li>
```
Comment (johngruen) [inline locales/en-US/experiments.ftl]: ditto

File: frontend/src/app/containers/ExperimentPage.js [hunk 4]
```diff
@@ -421,7 +421,7 @@ export class ExperimentDetail extends React.Component {
                   {this.renderLocaleWarning()}
                   {hasAddon && <div>
                     {!!introduction && <section className="introduction">
-                      <div data-l10n-id={this.l10nId('description')} dangerouslySetInnerHTML={createMarkup(introduction)}></div>
+                      <div data-l10n-id={this.l10nId('introduction')} dangerouslySetInnerHTML={createMarkup(introduction)}></div>
```
Comment (johngruen) [inline frontend/src/app/containers/ExperimentPage.js:424]: @clouserw so this was a bug. I realize this PR is for content, but this was broken.

File: locales/en-US/experiments.ftl (hunk 2 above)
Comment (ghost) [reply locales/en-US/experiments.ftl]: Good catch.  Please whack it on https://github.com/mozilla/testpilot/blob/master/content-src/experiments/no-more-404s.yaml#L29

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (johngruen) [reply locales/en-US/experiments.ftl]: @chuckharmston, any thoughts here?

File: locales/en-US/experiments.ftl (hunk 3 above)
Comment (ghost) [reply locales/en-US/experiments.ftl]: Regex missed them.  Please delete @ https://github.com/mozilla/testpilot/blob/master/content-src/experiments/universal-search.yaml#L27

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (johngruen) [reply locales/en-US/experiments.ftl]: FWIW, it appears to be in the german build n' stuff

File: frontend/src/app/containers/ExperimentPage.js (hunk 4 above)
Comment (ghost) [reply frontend/src/app/containers/ExperimentPage.js:424]: does look buggy!

File: frontend/test/app/containers/ExperimentPage-test.js [hunk 5]
```diff
@@ -144,7 +144,6 @@ describe('app/containers/ExperimentPage:ExperimentDetail', () => {
     subject.setProps({ hasAddon: true });
     // The measurements section is rendered twice, for responsiveness reasons.
     expect(findByL10nID('testingMeasurements0')).to.have.property('length', 2);
-    expect(findByL10nID('testingDescription')).to.have.property('length', 1);
```
Comment (ghost) [inline frontend/test/app/containers/ExperimentPage-test.js:147]: why not testingIntroduction?

File: frontend/src/app/containers/ExperimentPage.js (hunk 4 above)
Comment (johngruen) [reply frontend/src/app/containers/ExperimentPage.js:424]: Well it's fixed now...

File: frontend/test/app/containers/ExperimentPage-test.js (hunk 5 above)
Comment (johngruen) [reply frontend/test/app/containers/ExperimentPage-test.js:147]: @clouserw that exists a few lines up the page :)

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (ghost) [reply locales/en-US/experiments.ftl]: I don't know enough to say.  seems like it shouldn't be disappearing... :-/

**Review by ghost — APPROVED** [review]
r+wc

File: locales/en-US/experiments.ftl (hunk 1 above)
Comment (chuckharmston) [reply locales/en-US/experiments.ftl]: I have no idea where that field came from, but it isn't in the list of localized fields: https://github.com/mozilla/testpilot/blob/master/frontend/tasks/util.js#L1-L12

**Review by ghost — APPROVED** [review]

**Comment by johngruen** [comment]
hmm

**Review by ghost — APPROVED** [review]

**Pull request merged**